api/admin/controller.py

```diff
@@ -1560,6 +1560,9 @@ def _create_or_update_list(self, library, name, entries, collections, deletedEnt
         list.name = name
         membership_change = False
 
+        if deleteAll:
+            list.entries = []
```

Comment:
I think this might leave the CustomListEntry rows in the database, detached from their list. We may need to modify that relationship in the model so they get deleted. Otherwise this approach seems good.

Reply:
Also, membership_change should be true in this case.

Reply:
Can I just get every entry in the list and call `list.remove_entry` on it?

Reply:
Yeah, I think that would also work.

Reply:
Updated this.